src/libs/sys-util-rs/README.md

```diff
@@ -0,0 +1,23 @@
+# kata-sys-util
+
+This crate is a collection of utilities and helpers for 
+[Kata Containers](https://github.com/kata-containers/kata-containers/) components to access system services.
+
+It provides safe wrappers over system services, such as:
+- cgroups
+- file systems
+- mount
+- NUMA
+
+## Support
+
+**Platforms**:
+- x86_64
+- aarch64
```

Comment:
I don't see any architecture specific code in this crate, so is this comment accurate?

/cc @bpradipt, @Jakob-Naucke.

Reply:
removed